validator/validator.go

```diff
@@ -0,0 +1,61 @@
+// Package validator usage:
+
+// type Application struct {
+// 	Name      string `json:"name" validate:"required,printascii"`
+// 	Namespace string `json:"namespace"`
+// }
+
+// Definitions: []definition.Definition{
+// 	{
+// 		Method:   definition.Create,
+// 		Function: Handle,
+// 		Parameters: []definition.Parameter{
+// 			{
+// 				Source:    definition.Query,
+// 				Name:      "target1",
+// 				Operators: []definition.Operator{validator.Var("gt=0,lt=10")},
+// 			},
+// 			{
+// 				Source: definition.Body,
+// 				Name:   "app",
+// 				// Should we call this automatically?
```

Comment:
Is this still a question?

Reply:
After offline discuss, we will call it explicit here.